### DISCUSSION TITLE
Help me to resolve this error. "Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/plain". Strict MIME type checking is enforced for module scripts per HTML spec.""

### DISCUSSION THREAD

**Discussion by Luciferxgkp** [open-sauced-craftwork/browser-extension#30] [General]
![image](https://user-images.githubusercontent.com/76958987/226091151-15ff173f-eaa8-4e2d-97d9-bf7f5494533e.png)

**Comment by akash-singh8** [comment]
Can you share your folder structure?

**Reply by Luciferxgkp** [reply]
![Capture](https://user-images.githubusercontent.com/76958987/226092307-feea8aab-1d9f-4db4-a0b2-e884f46ac220.JPG)

**Reply by Inzamamdev** [reply]
Got the same error

**Reply by Luciferxgkp** [reply]
I did not change any thing . I Have just install and initialize the pakages as per this [post](https://dev.to/jacksteamdev/create-a-vite-react-chrome-extension-in-90-seconds-3df7)

**Reply by Inzamamdev** [reply]
Hey, @Luciferxgkp I just got the solution to this problem. You have to select the dist folder not the whole vite-project folder

**Reply by akash-singh8** [reply]
Yeah, you are right @456535688765

**Reply by Inzamamdev** [reply]
Thank you @akash-singh8 but I got a new error now can you help me with that

**Reply by Luciferxgkp** [reply]
Thanks man

**Comment by Inzamamdev** [comment]
Got an error for the service worker. The service worker is not loaded

**Reply by akash-singh8** [reply]
I guess this's because you haven't run the local server. Simply use `npm run dev`.

**Reply by Inzamamdev** [reply]
Thank you @akash-singh8 the error got resolved
